src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/Block/index.tsx

```diff
@@ -0,0 +1,320 @@
+  switchFlowNode,
+  updateFlowNode,
+  getLanguage,
+  getExpandedNodes,
+  setExpanded,
+  getDebugInfo,
+  editTriggers,
+  disconnectNode
+}) => {
+  const { nodeType } = node
+  const { currentLang, defaultLang } = getLanguage()
+
+  const handleContextMenu = e => {
+    e.stopPropagation()
+    e.preventDefault()
+
+    if (defaultLang && defaultLang !== currentLang) {
+      toast.info('studio.flow.cannotAddContent')
+      return
+    }
+
+    switchFlowNode(node.id)
+    contextMenu(
+      e,
+      <Menu>
+        <MenuItem
+          icon="trash"
+          text={
+            <div className={sharedStyle.contextMenuLabel}>
+              {lang.tr('delete')}
+              <ShortcutLabel light keys={['backspace']} />
+            </div>
+          }
+          intent={Intent.DANGER}
+          onClick={onDeleteSelectedElements}
+        />
+        <MenuItem
+          icon="duplicate"
+          text={<div className={sharedStyle.contextMenuLabel}>{lang.tr('copy')}</div>}
+          onClick={() => onCopySelectedElement(node.id)}
+        />
+        <MenuDivider />
+        {nodeType === 'trigger' && <MenuItem icon="edit" text={lang.tr('edit')} onClick={() => editTriggers(node)} />}
+        <MenuItem icon="minimize" text={lang.tr('studio.flow.disconnectNode')} onClick={() => disconnectNode(node)} />
+        {nodeType === 'router' ? (
+          <React.Fragment>
+            <MenuDivider />
+            <MenuItem text={lang.tr('studio.flow.chips')}>
+              <MenuItem
+                text={lang.tr('studio.flow.transition')}
+                onClick={async () => {
+                  await switchFlowNode(node.id)
+                  updateFlowNode({ next: [...node.next, defaultTransition] })
+                }}
+                icon="flow-end"
+              />
+            </MenuItem>
+          </React.Fragment>
+        ) : null}
+      </Menu>
+    )
+  }
+
+  const inputPortInHeader = !['trigger'].includes(nodeType)
+  const outPortInHeader = !['failure', 'router', 'success'].includes(nodeType)
+  const canCollapse = !['failure', 'router', 'success', 'listen'].includes(nodeType)
+  const hasContextMenu = !['failure', 'success'].includes(nodeType)
+
+  const debugInfo = getDebugInfo(node.name)
+
+  const renderContents = () => {
+    switch (nodeType) {
+      case 'action':
+        return <ActionContents node={node} editNodeItem={editNodeItem} />
+      case 'execute':
+        return <ExecuteContents node={node} updateFlowNode={updateFlowNode} switchFlowNode={switchFlowNode} />
+      case 'router':
+        return <RouterContents node={node} editNodeItem={editNodeItem} selectedNodeItem={selectedNodeItem} />
+      case 'say_something':
+        return (
+          <SaySomethingContents
+            node={node}
+            content={node.content}
+            defaultLang={defaultLang}
+            selectedNodeItem={selectedNodeItem}
+            currentLang={currentLang}
+          />
+        )
+      case 'trigger':
```

Comment:
I think the only thing that should have been brought back is the trigger to support users that are using it. Finally I think that we might want to use the approach you drafted a while ago with the blocks being dragged to nodes (sequential execution)

Reply:
I'm not sure to get your point here. Any user who enable NDU on master currently has access to Triggers, Say nodes, execute, listen, etc.  I brought back the styling because I can't stand to see these ugly nodes.

I think it would be a good idea to bring Albert nodes to existing bots (eg: the new execute, say something) without relying exclusively on standard nodes

![image](https://user-images.githubusercontent.com/42552874/98565423-9a4d2180-227b-11eb-9ad5-44456d8f3e23.png)

Reply:
My point is say and execute should be steps in a node (like we have in the standard node), some users are concerned about having to split their nodes.  Some of them have bots with more than 5000 nodes, splitting them would result in more than 25k ... making their flow totally unusable

Reply:
I agree, but that's out of the scope of this PR. The nodes are already splitted and actively being used. Only the design was changed so we can cleanup the code base. 

We'll find a better way to join them together once all nodes share the same base components

Reply:
OK . Max already started something in that direction